index.js

```diff
@@ -29,20 +33,104 @@ cp.parseBlock = function () {
 	return this.finishNode(node, "BlockStatement");
 };
 
+function strToModuleIdentifier(str) {
```

Comment:
By not supported I meant destructuring syntax. But ya this could be turned into a `let` or `const` list for sure.